tests/main/CMakeLists.txt

```diff
@@ -54,6 +54,8 @@ DEFINE_LAUNCH_TEST(hvdc_line_normal)
 
 DEFINE_LAUNCH_TEST(hvdc_line_inverse)
```

Comment:
Thanks for adding the hvdc_dangling case.
I think we should remove the hvdc_line_inverse one as it should be private.

Reply:
Should I wait for confirmation before removing the test for @rosiereflo or @marcochiaramello  ?

Reply:
If we have a complete test with both vsc and lcc, both normal and inverse (you just have to inverse the converterStations of one of the two lines), then there is no need for this one.
But let's wait for @marcochiaramello and @rosiereflo confirmation.

Reply:
yes, I think we can remove that case

Reply:
In fact I think you don't even need the normal test case either but just the hvdc_dangling that now covers everything

Reply:
It does not hurt to have a test that could cover cases we did not know about

Reply:
We should have one or 2 tests with the 4 possibilities inside (VSC-LCC)/(SIDE1-SIDE2). As long as this is covered I will be OK

Reply:
The hvdc_dangling covers them all as Guillaume did the inversion of one of the two lines

Reply:
@PerninGuillaume can you confirm that you removed the whole inverse case ? (it is a private case)

Reply:
Now, it is